doc/classes/NavigationMesh.xml

```diff
@@ -95,6 +95,7 @@
 		<member name="agent_radius" type="float" setter="set_agent_radius" getter="get_agent_radius" default="0.5">
 			The distance to erode/shrink the walkable area of the heightfield away from obstructions.
 			[b]Note:[/b] While baking, this value will be rounded up to the nearest multiple of [member cell_size].
+			[b]Note:[/b] The radius must be positive. If the radius is zero, a major correction step during the baking process will be disabled. Invalid outline overlaps and other precision errors of the input will not be fixed, removing unintended parts of the baking result.
```

Comment:
What **does** the "_removing unintended parts of the baking result._"? The sentence makes it sound like that is a direct consequence of "_[...] errors will not be fixed_", which sounds like a nice thing. Are you sure?

Reply:
if the radius is zero -> an auto-correction cannot run -> some parts of the final navigation mesh will be missing.

errors not being fixed sounds nice?

maybe like this?  
`[…] If the radius is zero, a major correction step during the baking process cannot run. This step is necessary to fix invalid outline overlaps and other precision errors of the input. Possible issues that occur if the step is skipped are: some obstacles are not being excluded correctly from the final navigation mesh, or an obstacle deletes the navigation mesh polygons.`

Reply:
I can work with this. Now the note is a bit too verbose, though? If I understand this correctly:
```suggestion
			[b]Note:[/b] The radius must be positive. If the radius is [code]0.0[/code], it won't be possible to fix invalid outline overlaps and other precision errors during the baking process. As a result, some obstacles may not be excluded correctly from the final navigation mesh, or may delete the navigation mesh's polygons.
```
or
```suggestion
			[b]Note:[/b] The radius must be positive. If the radius is [code]0.0[/code], it won't be possible to fix invalid outline overlaps and other precision errors during the baking process. As a result, some obstacles may delete the navigation mesh's polygons or may be included incorrectly in the final navigation mesh.
```
Or similar

Reply:
Changed one thing in the first one and pushed it